jobs/integr8ly/nightly-pds-heavy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+
+- job:
+    name: nightly-pds-heavy
+    project-type: pipeline
+    description: "Nightly pipeline for heavy PDS testing."
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H H(0-2) * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            YOURCITY=yourcity-hash
+            GH_CLIENT_ID=yourclientid
+            GH_CLIENT_SECRET=yourclientsecret
+            SELF_SIGNED_CERTS=true
+            RECIPIENTS=integreatly-qe@redhat.com
+            INSTALLATION_REPOSITORY=https://github.com/integr8ly/installation.git
+            INSTALLATION_BRANCH=master
+            TEST_SUITES_REPOSITORY=https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            TEST_SUITES_BRANCH=master
+            TO_DO=heavy
+    dsl: |
+        timeout(180) { ansiColor('gnome-terminal') { timestamps {
```

Comment:
Just wondering what is `ansiColor`?

Reply:
it is Jenkins plugin for making our logs more readable. @matskiv added it, he might explain what exactly it was configured.

Reply:
Yes, @trepel said, it's a jenkins plugin. It fixes how console output is displayed in jenkins.
Before:
![image](https://user-images.githubusercontent.com/1605799/52343676-0a7ca280-2a19-11e9-85c4-cc83eb17b127.png)
After:
![image](https://user-images.githubusercontent.com/1605799/52343698-1cf6dc00-2a19-11e9-94d2-07a220d57668.png)

Reply:
Nice thanks